http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -563,31 +589,9 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
       _ <- checkClusterPermission(userInfo, StopStartCluster, cluster, throw403 = true)
 
       now <- timer.clock.realTime(TimeUnit.MILLISECONDS)
-      _ <- internalStartCluster(userInfo.userEmail, cluster, Instant.ofEpochMilli(now))
```

Comment:
this email is never used in downstream functions, removing it from the method signature